lua/astrocommunity/editing-support/other-nvim/README.md

```diff
@@ -0,0 +1,10 @@
+# other-nvim
+
+Open alternative files for the current buffer.
+
+Supported Languages:
+
+- built-in: livewire, angular, laravel, rails, golang, python, react, rust, elixir
+- custom: clojure
```

Comment:
Any reason to add this to clojure?

Reply:
Not sure I understand what your asking.

To provide Clojure support to switch between src and test file was the only reason I submitted this PR.

Reply:
Maybe Clojure-specific opts should be extended in the Clojure pack? At the moment, any user will inherit them. Are they safe to be used in another language/framework?

Reply:
The other-nvim has many built-in languages which will be 'inherited' when anyone of those languages is used.

If this is a problem, the you shouldn't include this plugin at all.

Reply:
There is a difference between built-in languages, and then languages added by the community. One is the closest to a guarantee from the author. The other might break in the future.